application/dash_application/views/preqc_wgs.py

```diff
@@ -377,7 +390,56 @@ def generate_ploidy(df, colour_by, shape_by):
                 # TODO: add "Search Sample" input
 
                 # TODO: add "Show Names" dropdown
-                # TODO: add cut-off sliders
+                html.Label([
+                    "Reads Per Start Point:",
+                    core.Slider(
+                        id=ids["reads-per-start-point-slider"],
+                        min=0,
+                        max=50,
+                        step=1,
+                        marks={
+                            0: "0",
+                            5: "5",
+                            10: "10",
+                            15: "15",
+                            20: "20",
+                            25: "25",
+                            30: "30",
+                            35: "35",
+                            40: "40",
+                            45: "45",
+                            50: "50"
```

Comment:
Other reports use for loops to accomplish this

Reply:
Fancy. :wink: